test/GCHP/README.md

```diff
@@ -0,0 +1,68 @@
+# README for Integration Tests
+
+## Overview:
+
+This directory contains scripts to submit automated GCHP integration tests, which will test the following process:
+
+1. GCClassic run directory creation
+2. GCClassic code configuration and compilation
+3. GCClassic execution
+
+Integration tests are mostly intended to be run by GEOS-Chem developers, rather than by end users.
+
+## Files
+
+- `intTest.sh`
+  - Driver script to run GCHP integration tests
+- `intTestCreate.sh`
+  - Script to create GCHP run directories for integration tests
+- `intTestCompile.sh`
+  - Script to compile GCHP Classic executables
+- `intTestExecute.sh`
+   - Script to run GCHP Classic simulations
+- `commonFunctionsForTests.sh`
+  - Link to `../shared/commonFunctionsForTests.sh`, which contains global variables and functions for the integration test scripts.
+
+The integration tests are all short simulations (1 model hour).  Executables are compiled with all debugging options.  This should reveal any coding errors or run-directory configuration errors.
+
+## Running GCHP Integration Tests
+
+Before you submit any GEOS-Chem Classic integration tests, take a moment and make sure that:
+
+1. The `GCHP` superproject is checked out to the correct branch and commit.
+2. The `GEOS-Chem` submodule is checked out to the correct branch and commit.
+3. The `HEMCO` submodule is checked out to the correct branch and commit.
```

Comment:
Would it be better to say "2. Do `git submodule update --init --recursive' to ensure all submodules are checked out at the right branch and commit." That way we can include the other submodules (MAPL, etc.)

Reply:
I've clarified the instructions in commit e946a73a567926ad95696e7bc1d693ac13e24af3